docs/Administrator Guide/Geo Replication.md

```diff
@@ -264,22 +177,20 @@ gluster volume geo-replication <master_volume> \
     create [ssh-port <port>] push-pem|no-verify [force]
```

Comment:
Remove the optional slave_user signature.
[<slave_user>@] to <slave_user>@

Same in all other places below.

Reply:
Done